### PR TITLE
Implemented Hamburger Menu

### PR DESCRIPTION
**Pull request by room-projector**
Implemented Hamburger Menu for cleaner UI
image
- Advising dashboard and plan overview buttons hidden in this drawer.
- Log out/Log in now hidden in menu

Action bar now hidden in an edit plan drawer
image

Reviewers hidden in Reviewers drawer
image

Degree progress now a popup
image

Search shifted when degree progress open
image

Additional bug fixes.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +1,2 @@
-GENERATE_SOURCEMAP=false
\ No newline at end of file
+GENERATE_SOURCEMAP=false
+JWT_SECRET = “BL&qc6_gwP3=??!u”
```
Comment (JJamesWWang) [inline .env:2]: uh...this can't be right, surely?

**Comment by JJamesWWang** [comment]
man, y'all need to stop making changes that will break my tests 😭

File: .env (hunk 1 above)
Comment (room-projector) [reply .env:2]: LMAO no it's not; I'm putting .env on the gitignore. Nice catch.

Comment (room-projector) [reply .env:2]: Change pushed.

**Review by DenouementX — APPROVED** [review]
Looks good!

File: .env (hunk 1 above)
Comment (JJamesWWang) [reply .env:2]: We'll have to change the SECRET now, since it's been leaked; this will be preserved in git history.

Comment (JJamesWWang) [reply .env:2]: You should also delete the tracked .env file and commit that

Comment (room-projector) [reply .env]: Sounds good; tis done ^.^

Comment (JJamesWWang) [reply .env]: @QinyuanWu looks like you'll have to redistribute the new JWT_SECRET

Comment (room-projector) [reply .env]: I think it will expire in a few weeks so i think it's fine for now lol

Comment (room-projector) [reply .env]: In the future we'll have to xD

**Review by JJamesWWang — APPROVED** [review]
Looks good

**Review by robertzhidealx — APPROVED** [review]
I wouldn't personally prefer to put the plan controls in the menu along with others, but looks good otherwise.

**Review by alanz132 — APPROVED** [review]
Looks good

**Pull request merged**